lib/utils/user.js

```diff
@@ -7,7 +7,9 @@ import { MINUTE, string } from './';
 
 export const loggedInUser = _.once((): string | void => {
 	const userLink = document.querySelector('#header-bottom-right > span.user > a');
-	return userLink && !userLink.classList.contains('login-required') && userLink.textContent || undefined;
+	const words = userLink.href.split('/');
```

Comment:
There's actually a handy regex you can use to [extract username from `userLink.href.pathname`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/37511141d7e5571d837df826643f47487d5dc0d9/lib/utils/location.js#L55), sorry I didn't mention it in the original ticket.